src/discord_commands.rs

```diff
@@ -171,19 +172,19 @@ async fn before(ctx: &Context, msg: &Message, command_name: &str) -> bool {
 #[hook]
 async fn after(_ctx: &Context, _msg: &Message, command_name: &str, command_result: CommandResult) {
     match command_result {
-        Ok(()) => println!("Processed command '{}'", command_name),
-        Err(why) => println!("Command '{}' returned error {:?}", command_name, why),
+        Ok(()) => debug!("Processed command '{}'", command_name),
+        Err(why) => debug!("Command '{}' returned error {:?}", command_name, why),
```

Comment:
Should 176 be an error level log rather than a debug?

Reply:
Done.